src/components/LineTabPicker.tsx

```diff
@@ -58,7 +58,11 @@ export const LineTabPicker: React.FC<LineTabPickerProps> = ({ lines, tabState, t
                             {line.abbreviation}
                         </div>
                         <div className="label">
-                            {trains.length} {trains.length === 1 ? 'train' : 'trains'}
+                            {trains.length}{' '}
+                            <span className="wide-screen">
```

Comment:
Could you give this a class name that better describes what it is? Maybe `label-trains-text` or something.